.github/workflows/build.yml

```diff
@@ -5,53 +5,52 @@ name: build
+    branches: [main, feat/*]
   workflow_dispatch:
 
 jobs:
   build:
     name: Build and Test mxpy for ${{ matrix.os }}, python ${{ matrix.python-version }}
-    
+
     runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: [3.8, 3.11]
+        python-version: [3.11]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
-        python3 -m pip install --upgrade pip
-        pip3 install -r requirements.txt
-        pip3 install pytest
-    - name: Install libtinfo5
-      if: ${{ matrix.os != 'macos-latest' }}
-      run: |
-        sudo apt update
-        sudo apt install -y libtinfo5
-    - name: Set github_api_token
-      run: |
-        mkdir ~/multiversx-sdk
-        export PYTHONPATH=.
-        python3 -m multiversx_sdk_cli.cli config new test
-        python3 -m multiversx_sdk_cli.cli config set github_api_token ${{ secrets.GITHUB_TOKEN }}
-    - name: Setup test dependencies
-      run: |
-        python3 -m multiversx_sdk_cli.cli deps install testwallets
-        python3 -m multiversx_sdk_cli.cli deps install wasm-opt
-    - name: Run unit tests
-      run: |
-        export PYTHONPATH=.
-        pytest .
-    - name: Run CLI tests
-      run: |
-        python3 -m multiversx_sdk_cli.cli config set dependencies.vmtools.tag v1.4.60
-        cd ./multiversx_sdk_cli/tests
-        source ./test_cli_contracts.sh && testAll || return 1
-        source ./test_cli_dns.sh && testOffline || return 1
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip3 install -r requirements.txt
+          pip3 install pytest
+      - name: Install libtinfo5
+        if: ${{ matrix.os != 'macos-latest' }}
+        run: |
+          sudo apt update
+          sudo apt install -y libtinfo5
+      - name: Set github_api_token
+        run: |
+          mkdir ~/multiversx-sdk
+          export PYTHONPATH=.
+          python3 -m multiversx_sdk_cli.cli config new test
+          python3 -m multiversx_sdk_cli.cli config set github_api_token ${{ secrets.GITHUB_TOKEN }}
+      - name: Setup test dependencies
+        run: |
+          python3 -m multiversx_sdk_cli.cli deps install testwallets
+      - name: Run unit tests
+        run: |
+          export PYTHONPATH=.
+          pytest .
+      - name: Run CLI tests
+        run: |
+          python3 -m multiversx_sdk_cli.cli config set dependencies.vmtools.tag v1.4.60
+          cd ./multiversx_sdk_cli/tests
+          source ./test_cli_contracts.sh && testAll || return 1
```

Comment:
How come that we don't need to install clang / llvm, as well? No C smart contracts in our tests?

If so, we should add some tests for C contracts in the next PR.

Reply:
we only use rust contracts in our tests